src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleNoTrailingComma.java

```diff
@@ -9,20 +9,23 @@
 
 package com.puppycrawl.tools.checkstyle.checks.annotation.annotationusestyle;
 
-@SuppressWarnings({}) // violation
+@SuppressWarnings({}) // violation 'Annotation array values must contain trailing comma'
 public class InputAnnotationUseStyleNoTrailingComma
 {
-  @SuppressWarnings({"common"}) // violation
+  @SuppressWarnings({"common"}) // violation 'Annotation array values must contain trailing comma'
   public void foo() {
 
       /** Suppress warnings */
-      @SuppressWarnings({"common","foo"}) // violation
+      @SuppressWarnings(
+              {"common","foo"}) // violation 'Annotation array values must contain trailing comma'
```

Comment:
please try to not change java code (even formatting).
you can remove extra comments, like in this case and use `// violation below .....`

Reply:
Done. Code formatting has been rewritten to original form. Violation message has been rewritten accordingly.